go.mod

```diff
@@ -21,6 +21,7 @@ require (
 require (
 	cloud.google.com/go/compute v1.19.1 // indirect
 	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	github.com/chmike/domain v1.0.1 // indirect
```

Comment:
You might want to run `go mod tidy` to move this to a direct dependency instead of indirect.